src/test/java/io/orten/nano/OrganizationTest.java

```diff
@@ -38,7 +38,8 @@ public void before(){
 public void saveorganaization()
 
     {
-        Organization org= new Organization("67","microsoft", "NY", "nwerme", "mdf@gmail.com", "newpassword45", "brooklyn",
+        Organization org= new Organization("67","microsoft", "NY", "nwerme","0989769865"
+                , "mdf@gmail.com", "newpassword45", "brooklyn",
                 "154678", "very successful organisation");
         s.save(org);
```

Comment:
This test is not testing anything.
How do you know that org has actually been saved?
No manual intervention should be needed

Reply:
Leave as it for later refactoring